### PR TITLE
Hero stats and design updates

### PR DESCRIPTION
**Pull request by bradparker**
screen shot 2016-04-28 at 9 05 49 pm
screen shot 2016-04-28 at 9 05 57 pm
screen shot 2016-04-28 at 11 44 49 pm
screen shot 2016-04-28 at 11 44 56 pm
screen shot 2016-04-28 at 11 45 03 pm
screen shot 2016-04-28 at 11 45 06 pm

### CODE REVIEW

File: source/Routes/Home/index.js [hunk 1]
```diff
@@ -5,41 +5,30 @@ import { connect } from 'react-redux'
 import HomeLayout from '../../layouts/Home'
 
 import { fetchPage } from '../../store/actions/pages'
+import { fetchCampaign } from '../../store/actions/campaign'
 import { fetchCharities } from '../../store/actions/charities'
 import { fetchTeams } from '../../store/actions/teams'
-import { fetchResourceIntoStore, fetchCollectionIntoStore } from '../../utils/storeHelpers'
+
+const isFetched = ({ status } = {}) => status === 'fetched'
```
Comment (alexander-bobin) [inline source/Routes/Home/index.js]: I guess we could add these to utils at some point down the line

File: source/Routes/Home/index.js [hunk 2]
```diff
@@ -5,41 +5,30 @@ import { connect } from 'react-redux'
...
-      getState
-    }, {
-      fetcher: () => { fetchTeams(dispatch)('au-19283') },
-      collectionName: 'teams',
-      identifierKey: 'id'
-    })
+    unlessFetched(homePage, () => fetchPage(dispatch)('home')),
+    unlessFetched(campaign, () => fetchCampaign(dispatch)('au-19283')),
+    unlessFetched(charities, () => fetchCharities(dispatch)()),
+    fetchTeams(dispatch)('au-19283')
```
Comment (alexander-bobin) [inline source/Routes/Home/index.js]: We should probably add `au-19283` to a config somewhere (not this PR though)

File: source/components/CarouselDotNav/index.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    )
+  },
+  getIndexes (count, inc) {
+    var arr = []
+    for (var i = 0; i < count; i += inc) {
+      arr.push(i)
+    }
+    return arr
+  },
+  getListStyles () {
```
Comment (alexander-bobin) [inline source/components/CarouselDotNav/index.js]: Is there a benefit to having styles defined here as opposed to `styles.css`

Comment (bradparker) [inline source/components/CarouselDotNav/index.js]: Ha, caught out! This is taken from the slider's source code </lazy>

Comment (bradparker) [inline source/components/CarouselDotNav/index.js]: I still haven't written a for loop in a very long time.

File: source/Routes/Home/index.js (hunk 1 above)
Comment (bradparker) [inline source/Routes/Home/index.js]: `fetchUnless` would be happier in utils.

File: source/Routes/Home/index.js (hunk 2 above)
Comment (bradparker) [inline source/Routes/Home/index.js]: I'm actually thinking of putting in the initial state for the store..., move store init into `source/index.js`... and away we go

**Pull request merged**